tests/mockServer/mock.js

```diff
@@ -1165,21 +1165,18 @@ function putTags(req, res){
   const tagName = req.params.tag
   const projectName = req.params.project
 
-  const artifactForUpdate = artifacts.artifacts
+  const artifactList = artifacts.artifacts
```

Comment:
use the same name for const ( artifactsList or collectedArtifacts) for the same data